app/models/post.rb

```diff
@@ -954,7 +954,7 @@ def clean_fav_string!
     end
 
     def favorited_by?(user_id)
-      fav_string =~ /(?:\A| )fav:#{user_id}(?:\Z| )/
+      fav_string.split.include?("fav:#{user_id}")
```

Comment:
I don't know, I didn't consider the performance. I had wanted to avoid saying `!!(fav_string =~ /(?:\A| )fav:#{user_id}(?:\Z| )/)` to cast it to a boolean, since that's kind of ugly, but I'll change it back.